### ISSUE TITLE
Server-Side Rendering?

### ISSUE THREAD

**Issue by dts** [nolimits4web/Framework7-Vue#206]
I'd like to pre-render things (specifically, using NuxtJS) on the server for better rendering performance.  Out of the box, this doesn't work because `window` is referenced quite a bit, and with a shim in place, fails in other ways.  It's not obvious to me how to make this work.  Thanks!

**Comment by nolimits4web** [comment]
Not sure as well as F7 highly depends on window, document, etc.

**Comment by bencompton** [comment]
Might be able to leverage [jsdom](https://github.com/jsdom/jsdom) on the server and get it working that way.

**Comment by pi0** [comment]
This was the only thread I've found about SSR talks. After awesome idea of [ssr-window](https://github.com/nolimits4web/ssr-window) package, I've started nuxt-community/nuxt7#25 for SSR support. Still many areas including F7 and Template7 needs require `window` and `document` reference from this package.

**Comment by nolimits4web** [comment]
@pi0 yes. In last release it was included because Swiper and Dom7 relies on it. Will rework all references in F7 to window and document to next update
